app/src/androidTest/java/ch/epfl/sdp/musiconnect/database/MockDatabase.java

```diff
@@ -42,6 +42,11 @@ public Musician getDummyMusician(int index) {
     void addDoc(String collection, String docName, SimplifiedDbEntry entry) {
     }
 
+    @Override
+    public void addDoc(SimplifiedEvent simplifiedEvent, DbUserType userType) {
```

Comment:
Try to keep it package private and have a signature similar to addDoc above

Reply:
ok for private,
but there is only one collection for the events and the docName is generated by firestore in the addDoc function, so there is no docName to give for this function

Reply:
I would ave preferred that we keep one method per action but it's fine. We'll see about that later once we sort the server-side routines issue